packages/server/src/fhir/repo.test.ts

```diff
@@ -1254,4 +1254,19 @@ describe('FHIR Repo', () => {
       expect(cachedPatient.meta?.project).toEqual(project.id);
       expect(cachedPatient.gender).toEqual('unknown');
     }));
+
+  test('Handles resources with many entries stored in lookup table', async () =>
+    withTestContext(async () => {
+      const { repo } = await createTestProject({ withRepo: true });
+
+      const patient: Patient = {
+        resourceType: 'Patient',
+        link: [],
+      };
+      for (let i = 0; i < 64 * 1024 + 1; i++) {
```

Comment:
Maybe we can make a comment about why `64 * 1024` is selected here

Reply:
Also did this test for sure break? Since it's right at the limit

Reply:
Yep, it failed with `Error: bind message has 3 parameter formats but 0 parameters` without the change from this PR